tests/baseSetup.go

```diff
@@ -36,6 +37,9 @@ var (
 //One init function per package - initializes Rook infra and installs rook(if needed based on flags)
 func init() {
 	Env = objects.NewManifest()
+	if kv := os.Getenv("KUBE_VERSION"); kv != "" {
```

Comment:
this should be a flag that comes from the environment. lots of examples to follow from cmd/rook/main.go